Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -410,7 +410,7 @@
     
     /* Flag for checking of Strong matching is waiting on GAID fetch */
     private boolean performCookieBasedStrongMatchingOnGAIDAvailable = false;
-    private boolean isInstantDeepLinkPossible = false;
+    boolean isInstantDeepLinkPossible = false;
```

Comment:
I suggest that you keep it private, and make a "getter" to restrict access by other classes to the actual variable.